### DISCUSSION TITLE
Support for the experimental syntax 'exportDefaultFrom' isn't currently enabled (1:8)

### DISCUSSION THREAD

**Discussion by jeb2260** [babel/babel#11635] [Q&A] [answered]
Hi,

I'm trying to implement a pie chart for my react project as described here: https://canvasjs.com/react-charts/pie-chart/

Via command-line, I ran <strong>npm install canvasjs</strong> in my react project folder. 

Then I get the following error when running npm start: "<strong>Add @babel/plugin-proposal-export-default-from (https://git.io/vb4yH) to the 'plugins' section of your Babel config to enable transformation</strong>."

When looking up the error, I found this website: https://babeljs.io/docs/en/babel-plugin-proposal-export-default-from ,which says to run the following command:

<strong>npm install --save-dev @babel/plugin-proposal-export-default-from</strong>

I did run this command but am getting the same error. I've noticed some other people talk about this on Github: https://github.com/babel/babel/issues/7293

But I'm very new to Javascript and am bad at finding these dependencies and stuff. 

I noticed someone mention to change stuff in the <strong>babel.config.js<strong /> file as a solution, but I don't know where that file is located. I am on a Mac OS. Any help would be greatly appreciated

**Comment by nicolo-ribaudo** [comment, marked as answer]
Hi,
How are you running Babel? Directly with the Babel cli? Or are you using webpack? Or create-react-app?

**Reply by jeb2260** [reply]
Hi,

I used create-react-app

**Reply by nicolo-ribaudo** [reply]
create-react-app doesn't support `exportDefaultFrom` by default. You'll need to manually add it to your Babel config.

However, CRA hides the Babel config from you: you'll need to [eject](https://create-react-app.dev/docs/available-scripts/#npm-run-eject) in order to modify it.

Note that once you eject you can't go back: I suggest first evaluating if it's worth it, and if it isn't you should avoid using the "exportDefaultFrom" syntax:
```js
// Don't do this
export foo from "./foo";

// but this:
export { default as foo } from "./foo";
```

**Comment by boltcoder** [comment]
For anyone who's coming here from a non create-react-app world, but is using webpack (v4), I fixed my issue by ditching babel.config.json or babelrc and moved all the presets and plugins directly as option to the loader in the webpack config. Hope this helps
<img width="411" alt="Screenshot 2022-06-09 at 11 27 03 AM" src="https://user-images.githubusercontent.com/12605827/172774821-5a3cf067-eaaf-4965-affd-226c4b4442ab.png">
r
